include/brod.hrl

```diff
@@ -17,104 +17,55 @@
 -ifndef(__BROD_HRL).
 -define(__BROD_HRL, true).
 
--type kafka_key()                 :: kpro:key().
```

Comment:
if keeping them would cause some style check issue in your project
we can perhaps move the record definitions to a new header brod_record.hrl
so you don't have to include brod.hrl.